R/predictDirectedNetworks.R

```diff
@@ -0,0 +1,63 @@
+# predictDirectedNetworks.R
+
+source('./R/fetchData.R', echo=TRUE)
+source('./R/SyDButils.R', echo=TRUE)
+# ==============================================================================
+#' @title getSysInteractions
+#' @author Nada Elnour, \email{nada.elnour@@mail.utoronto.ca}
+#'
+#' \code{getSysInteractions} filter system by physically-interacting components
+#' and return their genetic interactions
+#'
+#' @param sysName A string specifying the full path to the excel sheet
+#' containing the system's components
+#' @param criterion A string, either "stringent" or "relaxed", specifying if
+#' only GGI between physical interactors should be selected.
+#'
+#' @return A dataframe of system components and either their GGI between
+#' physical interactors should be selected (iff \code{criterion} == "stringent")
+#' or all GGI of physical interactors (iff \code{relaxed} == "stringent")
+#'
+#' @importFrom dplyr filter
+#' @import utils
+#' @include SyDButils.R fetchData.R
+#' @importFrom stats complete.cases
+#' @examples
```

Comment:
A short explanation is required for each example of an exported function

Reply:
Fixed.